go.mod

```diff
@@ -25,24 +25,38 @@ require (
 	golang.org/x/sys v0.7.0
 )
 
+require (
+	github.com/google/go-github/v52 v52.0.0
```

Comment:
`tools` Go modules already uses `github.com/google/go-github/v39` and `github.com/google/go-github/v41`, so let's use one of those